src/workerd/api/node/zlib-util.h

```diff
@@ -1,9 +1,15 @@
 // Copyright (c) 2017-2022 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
+#include "src/workerd/jsg/jsg.h"
+#include "zlib.h"
 #include <kj/array.h>
```

Comment:
Can we keep the internal/external/system wise includes in an alphabetical order and within different groups?

Reply:
I thought clang-format would take care of this. I'll look into it.

Reply:
I fixed it manually this time, but we could definitely look into configuring clang-format to do this,.